cogs/pet.py

```diff
@@ -0,0 +1,65 @@
+import discord
+from discord.ext import commands
+
+import utils
+from config import app_config as config, messages
+
+from PIL import Image, ImageOps, ImageDraw
+import sys
+
+from io import BytesIO
+import requests
+
+config = config.Config
+messages = messages.Messages
+
+
+class Pet(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.cooldown(rate=5, per=20.0, type=commands.BucketType.user)
+    @commands.command(aliases=[])
+    async def pet(self, ctx, user: discord.Member = None):
+        if user is None:
+            user = ctx.author      
+        url = user.avatar_url_as(format='jpg')
+        response = requests.get(url)
+        avatarFull = Image.open(BytesIO(response.content))
+
+        frames = []
+        deformWidth = [-1, -2, 1, 2, 1]
+        deformHeight = [4, 3, 1, 1, -4]
+        width = 80
+        height = 80
+
+        for i in range(5):
+            frame = Image.new('RGBA', (112,112), (255,255,255, 1))
+            hand = Image.open('images/pet/' + str(i) + '.png')
```

Comment:
fstrings are much nicer IMO
```suggestion
            hand = Image.open(f"images/pet/{i}.png")
```